model/browser/events.yaml

```diff
@@ -52,3 +52,92 @@ groups:
           brief: A unique ID representing this particular metric instance.
           requirement_level: required
           examples: ["v3-1677874579383-6381583661209"]
+  - id: event.browser.user_action
+    stability: development
+    type: event
+    name: browser.user_action
+    brief: >
+      This event describes actions performed by the user such as click, scroll, zoom, resize, etc.
+    body:
+      id: browser.user_action
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: element
+          type: string
+          stability: development
+          brief: Target element tag name and it is obtained via `event.target.tagName`.
+          requirement_level: recommended
+          examples: ["button"]
+        - id: xpath
+          type: string
+          stability: development
+          brief: Target element xpath
+          requirement_level: recommended
+          examples: ["//*[@id='testBtn']"]
+        - id: user_action_type
+          type: enum
+          members:
+            - id: click.left
+              value: "click.left"
+              stability: development
+              brief: An element is left clicked by a user.
+            - id: click.right
+              value: "click.right"
+              stability: development
+              brief: An element is right clicked by a user.
+            - id: click.middle
+              value: "click.middle"
+              stability: development
+              brief: An element is middle clicked by a user.
+            - id: scroll
+              value: "scroll"
+              stability: development
+              brief: An element is scrolled by a user.
+            - id: zoom
+              value: "zoom"
+              stability: development
+              brief: An element is zoomed by a user.
+            - id: resize
+              value: "resize"
+              stability: development
+              brief: An element is resized by a user.
+            - id: keyboard.enter
+              value: "keyboard.enter"
+              stability: development
+              brief: An element is entered via keyboard by a user.
+            - id: keyboard.space
+              value: "keyboard.space"
+              stability: development
+              brief: A space is entered via keyboard by a user.
+            - id: other
+              value: "other"
+              stability: development
+              brief: User actions that are not listed above.
+          stability: development
+          brief: >
+            Type of interaction.
+            See enum [here](https://github.com/microsoft/ApplicationInsights-JS/blob/main/extensions/applicationinsights-clickanalytics-js/src/Enums.ts) for potential values we could add support for.
+          requirement_level: required
+          examples: ["click.right"]
+        - id: click.x
```

Comment:
I think the name click.* might be ambiguous and too specific at the same time. The browser API provides clientX/clientY, pageX/pageY, and screenX/screenY (see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSSOM_view/Coordinate_systems) for more details). The description here says "relative to the entire document", so I think click.x/click.y is intended to represent pageX/pageY. But given that there are multiple coordinate systems, shouldn't we be consistent and use the name that describes which coordinates are used?

Also, touch events have coordinates as well (also represented by pageX/pageY), so then these attributes could be used for both mouse and touch events.